contracts/v3/core/Module.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.7.0;
+
+// SPDX-License-Identifier: MIT
+
+abstract contract Module {
+
+    //Replaceable Modules
+    bytes32 constant BANK_MODULE = keccak256("bank");
+    bytes32 constant MEMBER_MODULE = keccak256("member");
+    bytes32 constant PROPOSAL_MODULE = keccak256("proposal");
+    bytes32 constant VOTING_MODULE = keccak256("voting");
+    bytes32 constant ONBOARDING_MODULE = keccak256("onboarding");
+    bytes32 constant FINANCING_MODULE = keccak256("financing");
+
+    //NonReplaceable Modules
+    bytes32 constant MANAGING_MODULE = keccak256("managing");
+
+    function isReplaceable(bytes32 moduleId) internal pure returns (bool) {
```

Comment:
why do you need that? with this logic you won't be able to either add a new adapter nor edit the existing ones no ?